cmd/openshift-install/testdata/agent/image/validations/sno_invalid_arch.txt

```diff
@@ -1,28 +1,22 @@
-# Verify a default configuration for the SNO topology for ARM architecture
+! exec openshift-install agent create image --dir $WORK
```

Comment:
Hey @pawanpinjarkar the issue is the original test case is using a x86 release payload, and you can't create a valid agent.aarch64.iso from that. Since this PR checks to see if the arch of the release payload matches the target arch, the old test case will no longer pass. Unfortunately, there isn't an upstream ARM64 release payload available in CI.

Reply:
This kind of problem seems to become more pressing, and removing tests doesn't seem a good strategy in the long term. It could be an acceptable approach for the current PR, as well as if we could start discussing (in a separate thread) on a possible way to support it somehow.

Reply:
@andfasano agree, happy to discuss offline. Unfortunately, there aren't a lot of options for multi-arch CI testing. We should be able to do validation in nightly's (if it isn't there already)